src/main.js

```diff
@@ -227,3 +222,30 @@ function addCommentToIdea(newComment, parentId) {
     }
   }
 }
+
+function deleteComment(event) {
+  var targetId = parseInt(event.target.id)
+  if (event.target.classList.contains("comment-delete")) {
+    for (var i = 0; i < savedComments.length; i++) {
+      if (targetId === savedComments[i].id) {
+        checkIdeasArray(savedComments[i].parentId, targetId)
+        savedComments[i].deleteFromStorage()
+        savedComments.splice(i, 1)
+        showIdeas(savedIdeas.sort(compareId));
+      }
+    }
+    modal.style.display = 'none'
+  }
+}
+
+function checkIdeasArray(id, targetId) {
```

Comment:
This is undoubtedly the most prolific piece of code written during this project. I am proud of how we tackled the challenge of only deleting the individual comment from the idea card it is associated with. We should all be very proud.